packs/data/spell-effects.db/spell-effect-chromatic-armor.json

```diff
@@ -56,6 +56,19 @@
                 "key": "Resistance",
                 "type": "{item|flags.pf2e.rulesSelections.chromaticArmor}",
                 "value": "5+5*floor((@item.level - 4) / 3)"
+            },
+            {
+                "key": "TokenLight",
+                "value": {
+                    "animation": {
+                        "intensity": 1,
+                        "speed": 3,
+                        "type": "pulse"
+                    },
+                    "bright": 20,
+                    "color": "#fff9a3",
+                    "dim": 40
+                }
             }
```

Comment:
Can we get a screenshot of this with a token and some background image to see if it's going to wash out everything?

Reply:
<img width="521" alt="Screen Shot 2023-01-19 at 11 57 15 AM" src="https://user-images.githubusercontent.com/1904698/213547002-806f04d0-5cee-483a-a6fd-8a086b25ebb3.png">